src/Microsoft.DotNet.Build.Tasks.Feed/GetBlobFeedPackageList.cs

```diff
@@ -26,6 +26,11 @@ public class GetBlobFeedPackageList : BuildTask
         [Output]
         public ITaskItem[] PackageInfos { get; set; }
 
+        /// <summary>
+        /// Specify max number of minutes that the lock of the feed container will be hold.
```

Comment:
Also, the description is not quite accurate.  It's the maximum wait time that the task will wait to acquire the lock, after which it will error out., not how long it will be held.